pages/Network_APIs/Live_Media_Distribution/Requirements_for_Live_Media_Distribution.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Requirements
+parent: Live Media Distribution
+grand_parent: Network APIs
+nav_order: 1
+has_children: false
+---
+
+{: .warning }
+This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Network Exposure and APIs Supporting Media Services and Applications**
+We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
+
+# Network Capabilities (Network Services) for Live Segmented Media Distribution
+
+## Quality of Service
+The following requirements are defined:
+- Ability to request different QoS profiles for individual data flows being distributed across a target service area
+- Ability to provision individual data flows for realiably low latency (distribution delay)
+- Delivery to endpoint (Application Media Server) may be identified by security/protocol/IP/port
```

Comment:
This is not a high-level requirement. It should be moved to the candidate solution.

Reply:
Alternatively, it could be moved nearer the end of the bulleted list and turned into a higher-level requirement: "The ability to identify the set of application data flows that fall within the scope of a particular QoS profile treatment".